.github/tests.yml

```diff
@@ -0,0 +1,36 @@
+name: Tests
+
+on: ['push', 'pull_request']
+
+jobs:
+    tests:
+        runs-on: ubuntu-latest
+        strategy:
+            fail-fast: true
+            matrix:
+                php: [7.3 7.4, 8.0]
```

Comment:
This is not correct.

Reply:
can you elaborate? which versions do you want there?

Reply:
Actually, can I transfer this repository to you? I don't plan to maintain this anymore.

Reply:
Me neither, I'm just random contributor here, I don't even use this package :( I wanted to test it out and saw it's not installable so I just fixed it.